engines/enginetest/testutils.go

```diff
@@ -27,6 +30,12 @@ func evalNilOrPanic(f func() error, a ...interface{}) {
 	nilOrpanic(f(), a...)
 }
 
+func assert(condition bool, a ...interface{}) {
+	if !condition {
+		fmtPanic(a...)
+	}
+}
+
```

Comment:
@petemoore, @gregarndt, I don't want to use the `assert` library from:
https://godoc.org/github.com/stretchr/testify/assert

It requires `t` be passed around.. and I'm too lazy for that... Honestly I'm quiet happy just doing a `panic()` it also gives me a stack trace... which is really what I want :)

Note, IMO, it's okay to define these non-exported helpers whenever we feel like it... This assert and nilOrPanic stuff is local to the `enginetest` package and can't be used outside it. I use them to keep the code concise otherwise the generic test cases I implement would simply be huge.

Note: also `panic` works from a go rutine.. `t` can only be called from the go rutine that called the `Test...` method.